Core/src/org/sleuthkit/autopsy/report/TableReportGenerator.java

```diff
@@ -899,23 +899,32 @@ public int compareTo(ArtifactData otherArtifactData) {
 
             } else if (BlackboardArtifact.ARTIFACT_TYPE.TSK_INTERESTING_FILE_HIT.getTypeID() == getArtifact().getArtifactTypeID()) {
                 String[] attributeDataArray = new String[3];
-                // Array is used so that the order of the attributes is 
-                // maintained.
+                // Array is used so that order of the attributes is maintained.
                 for (BlackboardAttribute attr : attributes) {
                     if (attr.getAttributeType().equals(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_SET_NAME))) {
                         attributeDataArray[0] = attr.getDisplayString();
                     } else if (attr.getAttributeType().equals(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_CATEGORY))) {
                         attributeDataArray[1] = attr.getDisplayString();
-                    } else if (attr.getAttributeType().equals(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_PATH))) {
-                        String pathToShow = attr.getDisplayString();
-                        if (pathToShow.isEmpty()) {
-                            pathToShow = getFileUniquePath(content);
-                        }
-                        attributeDataArray[2] = pathToShow;
                     }
                 }
+
+                attributeDataArray[2] = content.getUniquePath();
+                if (attributeDataArray[2].isEmpty()) {
+                    attributeDataArray[2] = getFileUniquePath(content);
+                }
```

Comment:
getFileUniquePath calls Content.getUniquePath, which should always return something. I believe lines 912-914 can be removed.